tests/functional/api/test_groups.py

```diff
@@ -209,3 +209,15 @@ def test_group_wiki(group):
     wiki.save()
     wiki.delete()
     assert len(group.wikis.list()) == 0
+
+
+def test_group_hooks(group):
```

Comment:
Unfortunately we don't have an EE license/docker container running for the functional tests so we will have to skip this for now to have them pass:

```suggestion
@pytest.mark.skip(reason="EE feature")
def test_group_hooks(group):
```

Reply:
I went ahead and applied this so that it makes it into tomorrow's scheduled release, thanks again @sugonyak!

Reply:
Awesome, thank you! I'm a bit AFK right now :)